### DISCUSSION TITLE
Change method

### DISCUSSION THREAD

**Discussion by camoredo** [chimurai/http-proxy-middleware#873] [Q&A]
Hi, would just like to ask if it's possible to change the method of the proxy request? For example, I wanted the request `GET /hello` to be sent as `POST` request to the target url. Thanks

**Comment by chimurai** [comment]
You'll need to modify to the request.

This might help you a bit: https://github.com/chimurai/http-proxy-middleware/blob/master/recipes/modify-post.md

**Reply by eyalrin** [reply]
Hi @chimurai — Have you found a way to do it in the end?
